fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -992,7 +975,7 @@ def count_unknowns(
                 f"Unexpected type of encoding {type(encoding)}, should be list or Encoding"
             )
         if unk_token is None:
-            unk_token = special_wrap_input(special_tokens["unk_token"])
```

Comment:
why do you prefer to hardcode it? can't it be different for different (sub) tokenizers?

Reply:
The special_tokens was also hard coded. 
We (I 😅 ) need to add such an argument to encode method (like we do with padding)